src/modules/UI/components/Buttons/style.js

```diff
@@ -14,7 +14,8 @@ export const rawStyles = {
   },
   buttonText: {
     fontFamily: THEME.FONTS.DEFAULT,
-    fontSize: 18
+    fontSize: 18,
+    lineHeight: 18
```

Comment:
which buttons were affected by this?

Reply:
The main buttons affected were the Export Transaction scene buttons, but this should ensure that a.) no more buttons have text getting cut off (top or bottom) and b.) that the button text is properly aligned vertically.